spark/src/test/scala/org/apache/comet/CometCastSuite.scala

```diff
@@ -838,8 +838,9 @@ class CometCastSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     castTest(generateTimestamps(), DataTypes.DoubleType)
   }
 
-  test("cast TimestampType to DecimalType(10,2)") {
-    castTest(generateTimestamps(), DataTypes.TimestampType)
```

Comment:
@andygrove This test case introduced in #356 looks incorrect. Could you please take a look?